files/js/cookies.ts

```diff
@@ -1,22 +1,22 @@
 /**
  * Set cookie.
- * 
- * @param {string} name Cookie name.
- * @param {string|boolean} value Cookie value. Defaults to boolean true.
+ * @param {Object} cookie setCookie function requires an object with the following properties:
+ * @param {string} cookie.name Cookie name.
+ * @param {string|boolean=} cookie.value Cookie value. Defaults to boolean true.
  */
-export const setCookie = (name: string, value: string|boolean = true): void => {
+export const setCookie = ({ name, value = true } : { name: string, value?: string|boolean }): void => {
 	const date = new Date();
 	date.setTime(date.getTime() + (365 * 24 * 60 * 60 * 1000));
 	document.cookie = name + '=' + value + '; expires=' + date.toUTCString() + '; path=/';
 }
 
 /**
  * Get cookie by name.
- * 
- * @param {string} name Cookie name.
+ * @param {Object} cookie getCookie function requires an object with the following properties:
+ * @param {string} cookie.name Cookie name.
  * @returns {string} Cookie value.
  */
-export const getCookie = (name: string): string => {
+export const getCookie = ({name}:{name: string}): string => {
```

Comment:
```suggestion
export const getCookie = ({ name }: { name: string }): string => {
```
I prefer spaces around the property names like you had in `setCookie()`